src/commands/bootstrap.ts

```diff
@@ -53,6 +53,29 @@ export function bootstrap({
         console.error(error);
       }
     }
+
+    if (reaction.emoji.name === '📌') {
+      try {
+        const users = await reaction.users.fetch();
+        // Get user that reacted with the pushpin emoji
+        const user = users.first();
+        const pinnedEmbed = new MessageEmbed()
+          .setColor('#0099ff')
+          .setTitle("Here's your pinned message buddy!")
+          .addFields(
+            { name: 'Author', value: reaction.message.author },
+            { name: 'Content', value: reaction.message.content }
```

Comment:
```suggestion
            { name: 'Content', value: reaction.message.content || "Embedded messages cannot be pinned."}
```
So I ran in to a similar issue with my personal bot (logging deleted message embeds). I created a fall-back value in case the `message.content` was empty (which is the case with embeds). Doing the same here should prevent the error.

Reply:
This would mean we can't support pinning embeds - but that is a feature we could implement later on if we wanted to look in to it.